teacher/templates/teacher_base.html

```diff
@@ -1,74 +1,168 @@
+          $('.js-hide-sidedrawer').on('click', hideSidedrawer);
+        });
+    </script>
+    <script>
+    $(document).ready(function(){
+      $(".intro").click(function(){
+        if ($(this).hasClass("intro")){
+            $(this).removeClass("intro").addClass("deintro");
+        }else if($(this).hasClass("deintro")){
+            $(this).removeClass("deintro").addClass("intro");
+        }
+      });
+    });
 
+    $(document).ready(function(){
+      var $rightpanel = $('#mui-panel');
+      $(".instafeedtrigger").click(function() {
+        if ($rightpanel.hasClass('rightpanelbody-hidden')) {
+            $rightpanel.removeClass('rightpanelbody-hidden').addClass('rightpanelbody-visible');
+        } else if ($rightpanel.hasClass('rightpanelbody-visible')){
+            $rightpanel.removeClass('rightpanelbody-visible').addClass('rightpanelbody-hidden');
+        };
+      });
+    });
+    </script>
+</head>
+<body class="hide-sidedrawer">
+    <header id="header" style="box-shadow: 0px 3px 35px 0px gray">
+      <div class="mui-appbar mui--appbar-line-height">
+        <div class="mui-container-fluid">
+          <a class="sidedrawer-toggle mui--visible-xs-inline-block mui--visible-sm-inline-block js-show-sidedrawer">☰</a>
+          <a class="sidedrawer-toggle mui--hidden-xs mui--hidden-sm js-hide-sidedrawer">☰</a>
+           <div style="float: right; cursor: pointer;" class="mui-dropdown mui-dropdown--left">
+              <a class="mui--text-left" data-mui-toggle="dropdown">
+                <i class="material-icons" style="color: white">more_vert</i>
+                <span class="mui-caret--left"></span>
+              </a>
+              <ul class="mui-dropdown__menu">
+                <li><a style="cursor: pointer;" class="instafeedtrigger">Instafeed</a></li>
+                <li><a href="{% url 'home-logout' %}">Log out</a></li>
+              </ul>
+          </div>
+        </div>
+        </div>
+      </div>
+    </header>
+    <div id="mui-panel" class="rightpanel rightpanelbody-hidden">
+      <h2 style="text-align: center; padding-bottom: 10px; font-size: 35px; color: #474747;">Instagram Feed</h2><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.2" />
+      <div id="instafeed">
+        <!-- INSTAFEED STUFF DOWN HERE !!!DO NOT CODE DOWN BELOW!!! -->
+      </div>
+    </div>
+    <div id="content-wrapper">
+      <div class="mui--appbar-height"></div>
+      <div class="mui-container-fluid">
+        <br>
+        {% block content %}
 
-</body>
+        {% endblock %}
+      </div>
+    </div>
+    <footer id="footer" style="background-color: #5c5c5c; text-align: center">
+      <div class="mui-container-fluid">
+        <!-- Links down below -->
+        <br>
+        <div style="text-align: center; vertical-align: text-bottom">
+            <br>
+            <br>
+        </div>
+      </div>
+      <h4>© Copyright 2019</h4>
```

Comment:
The footer of our main site currently has `2019 © Mission Bit. A 501(c)3 Non-Profit. ALL Rights Reserved.` - although it uses an expression to get the current year so it's always current (up to the last site update anyway, since it's statically built).

Reply:
This will be in a future PR (it is on the project requirements google doc as a requirement)